sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/README.MD

```diff
@@ -0,0 +1,9 @@
+Using the CoreWCF Postman Samples Collection.json within Postman
```

Comment:
We'll need to update this to follow the README conventions described in [the guidelines](https://azure.github.io/azure-sdk/general_documentation.html).